### PR TITLE
ENGDOCS-2202

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Consolidates settings pages into one page 

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Comment by dvdksn** [comment]
Is there a better way we can make these OS callouts? I find them hard to read in dark mode
image

File: content/desktop/settings.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 keywords: settings, preferences, proxy, file sharing, resources, kubernetes, Docker
-  Desktop, Mac
-title: Change Docker Desktop settings on Mac
+  Desktop, Linux, Mac, Windows
+title: Change your Docker Desktop settings
 aliases:
-- /docker-for-mac/mutagen-caching/
-- /docker-for-mac/mutagen/
-- /docker-for-mac/osxfs-caching/
-- /docker-for-mac/osxfs/
```
Comment (thaJeztah) [inline content/desktop/settings.md:10]: Do we have some other place where we maintain these redirects?

File: content/includes/experimental.md [hunk 2]
```diff
@@ -1,7 +0,0 @@
-Experimental features provide early access to future product functionality.
```
Comment (thaJeztah) [inline content/includes/experimental.md:1]: oh! interesting; thought we still would have this used somewhere, but maybe we don't 🤔

**Comment by dvdksn** [comment]
What about tables instead of lists?

**Pull request closed without merging**